pkg/controller/tuned/tuned_controller.go

```diff
@@ -110,15 +112,15 @@ func (r *ReconcileTuned) getTunedSecrets(tuned *tunedv1.Tuned) (map[string]bool,
 	}
 
 	sa := &corev1.ServiceAccount{}
-	err = r.client.Get(context.TODO(), types.NamespacedName{Namespace: saManifest.Namespace, Name: saManifest.Name}, sa)
+	err = r.cache.Get(context.TODO(), types.NamespacedName{Namespace: saManifest.Namespace, Name: saManifest.Name}, sa)
```

Comment:
What happens on a cache miss on `Get()`?

Reply:
You get stale information in exactly the same way you previously received with the split client and reconciliation loop will handle this once the cache is in sync.